src/main/java/com/amcglynn/myenergi/ZappiDaySummary.java

```diff
@@ -25,28 +25,26 @@ public ZappiDaySummary(List<ZappiHistory> dataPoints) {
         var importedJoules = new Joule();
         var evBoostJoules = new Joule();
         var evDivertedJoules = new Joule();
-        var evTotalJoules = new Joule();
 
         detectMissingDataPoints(dataPoints);
 
         for (var dp : dataPoints) {
-            solarGenerationJoules.add(dp.getSolarGeneration());
-            evBoostJoules.add(dp.getBoost());
-            evDivertedJoules.add(dp.getZappiDiverted());
-            evTotalJoules.add(dp.getZappiDiverted());
-            evTotalJoules.add(dp.getBoost());
-            exportedJoules.add(dp.getGridExport());
-            importedJoules.add(dp.getImported());
+            solarGenerationJoules = solarGenerationJoules.add(dp.getSolarGeneration());
```

Comment:
Defect here where the values were not being correctly added, resulting in each value being 0.0